src/plugins/intel_cpu/src/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -306,6 +307,62 @@ struct jit_uni_roi_pooling_kernel_f32 : public jit_uni_roi_pooling_kernel, publi
     }
 };
 
+namespace {
+struct RoiPoolingKey {
+    jit_roi_pooling_params refParams;
+
+    size_t hash() const;
+    bool operator==(const RoiPoolingKey& rhs) const;
+};
+
+size_t RoiPoolingKey::hash() const {
+    using namespace dnnl::impl;
+    using namespace dnnl::impl::primitive_hashing;
+
+    size_t seed = 0;
+
+    seed = hash_combine(seed, refParams.mb);
+    seed = hash_combine(seed, refParams.c);
+    seed = hash_combine(seed, refParams.nb_c);
+    seed = hash_combine(seed, refParams.ih);
+    seed = hash_combine(seed, refParams.iw);
+    seed = hash_combine(seed, refParams.oh);
+    seed = hash_combine(seed, refParams.ow);
+    seed = hash_combine(seed, refParams.alg);
+    seed = hash_combine(seed, refParams.src_prc.getPrecVal());
+    seed = hash_combine(seed, refParams.dst_prc.getPrecVal());
+    seed = hash_combine(seed, refParams.spatial_scale);
+    seed = hash_combine(seed, refParams.pooled_h);
+    seed = hash_combine(seed, refParams.pooled_w);
```

Comment:
I see that `nb_c_blocking` and `c_block` params are missed in seed computation. Is that expected?

Reply:
Yes, because I think the values of `nb_c_blocking` and `c_block` are determined in ``reatePrimitive()` according to the platform we use, and will not be changed during inference. Is it right?

Reply:
Yes, you are right that these parameters are fully static now. However I want to avoid satiation when someone will decide to make there parameters dependent on some non-trivial logic, but will forget to update hashing routine. Given that this is just 2 integer numbers hashing time increase won't be significant. 
Otherwise these parameters should be hidden as Executor private fields.

Reply:
Thank you @dmitry-gorokhov for the explanation. I added them to hash calculating.